lib/services/android-project-service.ts

```diff
@@ -179,25 +184,25 @@ class AndroidProjectService implements IPlatformProjectService {
 		this.$logger.info("Copying %s", projDir);
 		shell.cp("-Rf", projDir, destDir);
 
-		var targetDir = path.join(destDir, path.basename(projDir));
+		let targetDir = path.join(destDir, path.basename(projDir));
 		// TODO: parametrize targetSdk
-		var targetSdk = "android-17";
+		let targetSdk = this.getTarget(projDir).wait() || "android-17";
```

Comment:
Why do we need this change here? This will update only `project.properties` file